### DISCUSSION TITLE
Add BCC (blind carbon copy) field to the Send Email operation in Flows

### DISCUSSION THREAD

**Discussion by cs1m0n** [directus/directus#15913] [Feature Requests]
As far as I see from [#5563](https://github.com/directus/directus/pull/5563) and from what I've tested the MailService, the API supports BCC recipients, so what I think it's missing is only the field in the Send Email operation settings.

**Comment by azrikahar** [comment]
> so what I think it's missing is only the field in the Send Email operation settings.

Seems like it! Should be a similar "add new `bcc` option field" approach like #15829 does for templates. That said, BCC would probably still be justifiable as a dedicated option below the `To` field, but I think when more requests props up to add support for every other nodemailer options, maybe it should be implemented as a repeater instead for added flexibility while not clogging up the operation with too many options.

**Reply by rijkvanzanten** [reply]
> maybe it should be implemented as a repeater instead for added flexibility while not clogging up the operation with too many options.

The total number of options for an email is still pretty limited, so I'm down to just make fields for them individually. That being said, it might be a good idea to put some of those options under "Advanced" so we don't clog up the main options too much with these power user features 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
